Liker.Social/chains.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Liker.Social",
+  "chains": [
+    {
+      "name": "likecoin",
+      "address": "likevaloper1mztweu8y2lazpapfgtqmadxaqaapyasv7nhexk",
+      "restake": {
+        "address": "like1pp52z2a6wkuughsrxzy2cl2wldhcnyzvjcj7lv",
+        "run_time": "every 1 hour",
+        "minimum_reward": 5000000000
+      }
+    },
```

Comment:
Just need to remove this trailing comma otherwise the JSON is invalid. I'll sort in the morning if you haven't already, not at a laptop right now

Reply:
Done!, I made a stupid mistake, Thanks for your review!